.github/workflows/examples.yml

```
@@ -40,6 +40,16 @@ jobs:
     timeout-minutes: 60
     steps:
       - uses: actions/checkout@v2
+      - name: Get yarn cache directory path
+        id: global-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
```

Nit: This is already getting deprecated now, should we find an alternative?

https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs

```suggestion
        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
```